libs/blocks/caas-config/caas-config.js

```diff
@@ -36,13 +43,15 @@ const updateObj = (obj, defaultObj) => {
 
 const isValidUuid = (id) => /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/.test(id);
 
-const getHashConfig = () => {
+const getHashConfig = async () => {
```

Comment:
I don't see an await in this function, `async` needed?

Reply:
`getHashConfig` now can either be async (for v2 hashes) or sync (for v1 hashes).  I think for clarity, since it's possible that it will be async this function should be marked as async?  Otherwise you have to just know.  Thoughts?

Reply:
Nevermind, am updating the code to show why it's async.